src/mono/mono/mini/llvm-runtime.cpp

```diff
@@ -3,25 +3,13 @@
 
 #include <glib.h>
 
-#ifdef HOST_WASM
-#include <emscripten.h>
-#endif
-
 extern "C" {
 
 void
 mono_llvm_cpp_throw_exception (void)
 {
 	gint32 *ex = NULL;
 
-#ifdef HOST_WASM
-		EM_ASM(
-			   var err = new Error();
-			   console.log ("Throw stacktrace: \n");
-			   console.log (err.stack);
-			   );
-#endif
-
```

Comment:
What are these changes for?

Reply:
They were added accidently by 997dc08f691ee0246990a21c73850a041b80b800.